perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -45,11 +45,21 @@
 from perfkitbenchmarker import vm_util
 from perfkitbenchmarker.sample import PercentileCalculator  # noqa
 
+
 flags.DEFINE_enum('storage', benchmark_spec_class.GCP,
                   [benchmark_spec_class.GCP, benchmark_spec_class.AWS,
                    benchmark_spec_class.AZURE],
                   'storage provider (GCP/AZURE/AWS) to use.')
 
+flags.DEFINE_string('object_storage_gcp_region', 'us-central1',
+                    'Region for GCP object storage.')
+
+flags.DEFINE_string('object_storage_aws_region', 'us-east-1',
```

Comment:
The only difference between these 3 flags is their default values. Could you merge these 3 flags and use a dict to map object storage providers to regions?